app/src/main/java/com/koalatea/sedaily/feature/bookmarks/BookmarksFragment.kt

```diff
@@ -96,16 +100,16 @@ class BookmarksFragment : BaseFragment() {
         emptyStateContainer.visibility = View.GONE
         epoxyRecyclerView.visibility = View.GONE
```

Comment:
Also change the visibility of `epoxyRecyclerView.visibility `'s parent `SwipeRefreshLayout` instead

Reply:
Done.